pkg/registry/registry.go

```diff
@@ -44,6 +42,8 @@ const (
 	AntreaEnterpriseID uint32 = 55829
 	// IANAEnterpriseID is the enterprise ID for IANA Information Elements
 	IANAEnterpriseID uint32 = 0
+	// Enterprise ID for reverse Information Elements
```

Comment:
nit: move const and global variable to up top
Usually structs are defined after constants and global variables.